Vagrantfile

```diff
@@ -70,6 +75,10 @@ Vagrant.configure("2") do |config|
     config.vm.provision "prepare", type: "shell", path: "installer/vagrant/%s" % prepare_script
 
     # Provision LibreTime
+    if config.vm.box == "fujimakishouten/debian-buster64"
+        installer_args = installer_args + "--distribution=debian --release=buster"
```

Comment:
You could also pass these to the provison_libretime function call like the centos box already does...

```Vagrant
    provision_libretime(os, "debian.sh", installer_args + "--distribution=debian --release=buster")
```

This would work but the proper way to do this is probably to get the install script to get autodetection working.

Reply:
Ah yes, I'll do it like that then. The auto detection will be fixed once Buster is released - it will then be assigned an ID in `/etc/os-release`